.github/workflows/test.yml

```diff
@@ -19,66 +19,87 @@ jobs:
           - beta
           - nightly
           - 1.56.1
+    env:
+      RUSTFLAGS: "-C target-cpu=native -C opt-level=3"
+      ROARINGRS_BENCH_OFFLINE: "true"
 
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout roaring-rs
+        uses: actions/checkout@v2
 
-      - uses: actions-rs/toolchain@v1
+      - name: Checkout benchmark datasets
+        uses: actions/checkout@v2
+        with:
+          repository: "RoaringBitmap/real-roaring-datasets"
+          path: "benchmarks/real-roaring-datasets"
```

Comment:
Ok, so this makes sure that the `real-roaring-datasets` repository is cloned before we start running the benchmarks. I wasn't aware that the CI was offline, good catch!

This is probably to avoid crypto miners... I don't know.

Reply:
I added vendored ssl as you suggested, I suppose windows machines might not be able to link it (?). I still prefer checking out the repo explicitly in CI. It puts the network fetch at the beginning. I've also added fetch steps before compiling begins for the same reason.

Now if theres a network failure it should always happen in the first ~30 sec or so of CI